controllers/machine.go

```diff
@@ -180,8 +181,16 @@ func (mrc *machineReconcileContext) createTemplate(hardware *tinkv1.Hardware) er
 		return fmt.Errorf("failed to generate imageURL: %w", err)
 	}
 
+	metadataIP := os.Getenv("TINKERBELL_IP")
+	if metadataIP == "" {
+		metadataIP = "192.168.1.1"
+	}
+
+	metadataURL := fmt.Sprintf("http://%s:50061", metadataIP)
```

Comment:
Should the port and/or full URL be overridable too? (or are these not configurable in CAPT)

Reply:
Eventually, probably. Wanted to put a little more thought into how to expose the configurability before exposing it, and setting TINKERBELL_IP was already a pre-requisite step needed.